### DISCUSSION TITLE
The code ran too slow when I used "Multiprocessing" with emcee run

### DISCUSSION THREAD

**Discussion by kittipongwangnok** [phoebe-project/phoebe2#964] [General]
I just wondering when I used this command on the HPC, then the code was running too slow:

phoebe.multiprocessing_on()
print(phoebe.multiprocessing_get_nprocs())

The code seems to use the CPU core from HPC, but It is not processed too fast. How can we solve this problem?

**Comment by kecnry** [comment]
How many processors does phoebe see (what is the output from the print statement you mentioned)?  Do all of those processors get used?

If multiprocessing isn't working well on that particular setup for you, then you can try MPI as well (which has wider support in phoebe than multiprocessing, but can be a little more difficult to get set up).
